polaris.shopify.com/pages/api/search/v0/index.tsx

```diff
@@ -53,7 +56,7 @@ const getSearchResults = (query?: string) => {
       title,
       description = '',
       category = '',
-    } = pages[slug].frontMatter;
```

Comment:
The cast here is necessary since `satisfies` on the site JSON data structure forgets the type it satisfied. And since we currently don't have a `status` on any of the components, it thinks that field doesn't exist. Re-casting it back to `FrontMatter` reminds that `status?: string` does in fact exist.